donkeycar/parts/lidar.py

```diff
@@ -13,6 +14,187 @@
 from donkeycar.utils import norm_deg, dist, deg2rad, arr_to_img
 from PIL import Image, ImageDraw
 
+
+CLOCKWISE = 1
+COUNTER_CLOCKWISE = -1
+
+
+class RPLidar2(object):
+    '''
+    Adapted from https://github.com/Ezward/rplidar
+    '''
+    def __init__(self,
+                 min_angle = 0.0, max_angle = 360.0,
+                 min_distance = sys.float_info.min, max_distance = sys.float_info.max,
+                 forward_angle = 0.0,
+                 angle_direction=CLOCKWISE,
+                 debug=False):
+        
+        self.lidar = None
+        self.port = None
+        self.on = False
+        
```

Comment:
Too many blank lines (pep-8).

Reply:
I did a pep-8 on this code and got 321 lines of output.  "name 'e' does not conform to snake-case"  PEP-8 was written by an abuser.  Create a language that depends on invisible characters, then write tools that say you should be able to see the invisible characters. I've gone back and put one space between methods in classes and two spaces between classes.

Reply:
👍